bahamut.go

```diff
@@ -155,6 +179,11 @@ func (b *server) ProcessorForIdentity(identity elemental.Identity) (Processor, e
 	return b.processors[identity.Name], nil
 }
 
+func (b *server) CustomHandlers() map[string]http.HandlerFunc {
+
+	return b.customRoutesHandlers
```

Comment:
I don't think this api is needed

Reply:
that's how I retrieve them .. I don't think I have access to them otherwise in the function. These functions are not part of an object.

Reply:
Can it be private?